engine/swarm/admin_guide.md

```diff
@@ -262,3 +262,14 @@ The `--force-new-cluster` flag puts the Docker Engine into swarm mode as a
 manager node of a single-node swarm. It discards swarm membership information
 that existed before the loss of the quorum but it retains data necessary to the
 Swarm such as services, tasks and the list of worker nodes.
+
+### Re-joining a previously failed node
+
+If a node becomes unavailable, it cannot communicate with the rest of the swarm.
+If access to that node is restored, it will re-join the swarm, but it will
+function as a completely new node. In fact, when it first re-joins, its current
```

Comment:
I agree. Let's not say that re-joins the swarm, because according to our terminology, it never really left; it was just unreachable for a period of time. During that time, its workload would have been rescheduled elsewhere.